app/views/core/RootView.coffee

```diff
@@ -201,10 +201,11 @@ module.exports = class RootView extends CocoView
       #console.log 'Saved language:', newLang
 
   isOldBrowser: ->
-    if $.browser
+    if features.china and $.browser
```

Comment:
Do we use this function for other area?

Reply:
nick says no

Reply:
and i serach through source code and find none